examples/encrypted-events/Makefile

```diff
@@ -0,0 +1,26 @@
+all: build test
+
+install:
+	# Install local deps without relying on the workspace lockfile.
+	# Use npm ci when a lockfile is present; otherwise fall back to npm install without writing a lockfile.
+	if [ -f package-lock.json ] || [ -f npm-shrinkwrap.json ]; then \
+		npm ci; \
```

Comment:
We're using `pnpm`. See how Makefiles of other examples are doing it.

Also don't forget to add `examples/encrypted-events` package to pnpm-workspace.yaml.